doc/vhpidirect/examples/quickstart.rst

```diff
@@ -52,6 +54,70 @@ for further details.
 Since either C sources or pre-compiled ``.o`` objects can be added, in C/C++ projects of moderate complexity, it might
 be desirable to merge all the C sources in a single object before elaborating the design.
 
+.. _COSIM:VHPIDIRECT:Examples:wrapping:
+
+Wrapping ghdl_main
+******************
+
+.. _COSIM:VHPIDIRECT:Examples:quickstart:wrapping:basic:
+
+:cosimtree:`basic <vhpidirect/quickstart/wrapping/basic>`
+---------------------------------------------------------
+
+Instead of using GHDL's own entrypoint to the execution, it is possible to wrap it by providing a custom ``main``
+function. Upon existence of ``main``, execution of the simulation is triggered by calling ``ghdl_main``.
+
+This is the most basic example of such usage. ``ghdl_main`` is declared as ``extern`` in C, and arguments ``argc`` and
```

Comment:
Suggestion:

providing a custom program entrypoint (``main`` function), wherein the execution of the simulation....

_Below_ is the most basic example...

Reply:
I agree with the first part. However, "Below" is not exactly correct, because "This" means the section itself. That is, the header is `:cosimtree:basic <vhpidirect/quickstart/wrapping/basic>` and "this" means that example, not the code block below. I rewrote it to *"This example shows the most basic of such usage"*, but it sounds weird to me. What do you think?